db/fixtures/miq_product_features.yml

```diff
@@ -629,7 +633,7 @@
   - :name: View
     :description: View Cloud Object Store Containers
     :feature_type: view
-    :identifier: cloudobject_store_container_view
```

Comment:
I'm debating dropping this rename.  It's easy enough to workaround for now by changing the UI to assert_privileges on the old inconsistent `cloudobject` name.

The alternative is a database migration that looks up the existing feature, renames it and verifies any existing miq_user_roles with this feature still have it after the rename.

Currently, the existing product feature seeding will seed from the yml, adding the new feature, but will end up deleting the old named feature, causing any user roles to lose it.

Reply:
cc @kavyanekkalapu As we discussed... I'm ok with doing the rename in a week or so to see if there are other renames you find while doing development... or I can do the migration now.  What do you think?

Reply:
i think we can wait for a week @jrafanie , i will also see of there are any other identifiers i can find.

Reply:
:+1: for avoiding a db migration, so we can backport properly...then have a separate PR for the renames + db migration.